### DISCUSSION TITLE
Problemes with  ALVideoPlayer

### DISCUSSION THREAD

**Discussion by ossamalimon** [MagicFoundation/Alcinoe#210] [Q&A]
Hi ; 

i do  all thing to work with  ALVideoPlayer , and it s work fine in my emulator ; But when  i install it in my phone i get  this :  
![2476edf1-3b17-4b7b-a2a0-31f423369017](https://user-images.githubusercontent.com/64082995/129488778-a75f90ff-bff2-4112-8b57-cb1fbcca1212.jpg)
 the sound works fine but no video image .
Any idea for the solutions ? 

and thnx first .

**Comment by eilersn** [comment]
I have a similar problem, but I don't even get sound, just a little text like yours in a corner.
